cli/commands/project_sync.go

```diff
@@ -97,56 +97,60 @@ func (cmd *ProjectSync) RunStart(ctx *cli.Context) error {
 	}
 
 	// Determine the working directory for CWD-sensitive operations.
-	if workingDir, err := cmd.DeriveLocalSyncPath(cmd.Config, ctx.String("dir")); err == nil {
-		// Determine the volume name to be used across all operating systems.
-		// For cross-compatibility the way this volume is set up will vary.
-		volumeName := cmd.GetVolumeName(ctx, cmd.Config, workingDir)
-
-		switch platform := runtime.GOOS; platform {
-		case "linux":
-			cmd.out.Verbose.Printf("Setting up local volume: %s", volumeName)
-			cmd.SetupBindVolume(volumeName, workingDir)
-		default:
-			cmd.out.Verbose.Printf("Starting sync with volume: %s", volumeName)
-			cmd.StartUnisonSync(ctx, volumeName, cmd.Config, workingDir)
-		}
-	} else {
-		cmd.out.Error.Fatal(err)
+	var workingDir, err = cmd.DeriveLocalSyncPath(cmd.Config, ctx.String("dir"))
+	if err != nil {
+		return cmd.Error(err.Error(), "SYNC-PATH-ERROR", 12)
```

Comment:
Hm, maybe we should define cmd.EXTERNAL_ERROR_CODE and so on for more clarity about the numbers? I realize that's outside immediate scope but reading this out of the context of the other PR this immediately jumps out at me as "why is this a number 12?"

Reply:
I think some constants for standard error codes are probably a good idea. We can tackle that in another PR